src/hooks/useEditOrder.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Order } from "../types/Order";
+import { QueryClient, useMutation, useQuery } from "@tanstack/react-query";
+import { doc, setDoc } from "firebase/firestore";
+import { db } from "../Firebase";
+
+const useEditOrder = (queryClient: QueryClient) => {
+    return useMutation({
+        mutationFn: async (order: Order) => {
+            await setDoc(doc(db, "clients", order.id), {
+                "Boxes for AFAC": order.boxesForAFAC,
```

Comment:
One comment would be to use a central definition for the translation between Firebases document field names and the javascript object. There's just the same translation in useEditOrder and useOrders now. I honestly can't think of any clean syntax suggestions though. There's only two hooks for now, so it's not too tedious, but it might become tedious to maintain the parallel translations for more hooks.

Reply:
It would be easier if the database keys were the same as the object keys but this is the way the database was when I was given it. If we want to change it, we have to update every database entry. We can keep it in mind for future refactoring but lets leave it for now.